spec/features/admin/admin_can_activate_and_retire_items_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.